src/Buildalyzer.Workspaces/Buildalyzer.Workspaces.csproj

```diff
@@ -19,8 +19,8 @@ ToBeReleased
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.CodeAnalysis.CSharp.Workspaces" Version="4.4.0" />
-    <PackageReference Include="Microsoft.CodeAnalysis.VisualBasic.Workspaces" Version="4.4.0" />
+    <PackageReference Include="Microsoft.CodeAnalysis.CSharp.Workspaces" Version="4.10.0" />

```

Comment:
Did you try this one (`[4,)`) too? Did this work as well?

Reply:
It does not work with `[4,)`, version `4.0.1` gets installed and fails to build a project with C#12 features